k8s/README.md

```diff
@@ -1,6 +1,6 @@
 ## Kubernetes Files for Versions 2.1.1 and Up
 
-Detailed instructions for installing and upgrading the Scalyr agent in kubernetes can be found [here](https://app.scalyr.com/help/install-agent-kubernetes).
+Detailed instructions for installing and upgrading the Scalyr Agent in Kubernetes can be found [here](https://app.scalyr.com/help/install-agent-kubernetes).
```

Comment:
Right under the first line (the Kubernetes Files header), let's have this text:

This directory contains the manifest files for running the Scalyr Agent on your Kubernetes cluster.  Specifically, it defines the service account and DaemonSet required to run the Scalyr Agent in the `scalyr` namespace.  You must create a secret key and configmap before creating the service account and DaemonSet (see instructions below).

Prior to the Scalyr Agent 2.1 release, the Scalyr Agent K8s instructions created objects in the default namespace.  If you used these instructions prior to Scalyr Agent 2.1, you must either use the manifest files in the <link to default namespace directory> or perform the upgrade instructions documented at <link to where the instructions will be on the Scalyr website>

The follow up with the text you already have here (including the link to detailed instructions and the intended way to use the files)